### ISSUE TITLE
Add millisecond option for date_diff

### ISSUE THREAD

**Issue by therapon** [partiql/partiql-lang-kotlin#204]
It would be nice to have the option to use date_diff that outputs the time difference in milliseconds.
Ex: `DATE_DIFF(milliseconds, my_timestamp_field, UTCNOW())`

**Comment by dlurton** [comment]
Is this SQL-92?

**Comment by therapon** [comment]
> Is this SQL-92?

I do not think so. I believe SQL92 has `intervals` and relies on `extract` to grab scalars from an interval. `date_diff` seems to be a common date related function across DB implementations. 

I do not recall the source for adding this issue though (ask from someone or "coverage" for `date_diff` not sure)
